categories/css.json

```diff
@@ -70,5 +63,147 @@
       "learn",
       "visual"
     ]
+  },
+  {
+    "name": "postcss-preset-env",
+    "description": "A PostCSS plugin to convert modern CSS into something most browsers can understand.",
+    "url": "https://preset-env.cssdb.org/",
+    "free": true,
+    "tags": [
+      "postcss",
+      "css",
+      "preprocessor",
+      "nesting",
+      "matches"
+    ]
+  },
+  {
+    "name": "1 Line Layouts",
+    "description": "10 Modern CSS layout and sizing techniques in a single-line of code.",
+    "url": "https://1linelayouts.glitch.me/",
+    "free": true,
+    "tags": [
+      "css",
+      "layout",
+      "tutorial",
+      "flexbox",
+      "grid"
+    ]
+  },
+  {
+    "name": "CSS Houdini Experiments",
+    "description": "Experiments with the latest CSS Houdini spec.",
+    "url": "https://css-houdini.rocks/",
+    "free": true,
+    "tags": [
+      "css",
+      "houdini",
+      "demo"
+    ]
+  },
+  {
+    "name": "Is Houdini ready yet‽",
+    "description": "Browser support chart showing current support for CSS technologies under development.",
+    "url": "https://css-houdini.rocks/",
+    "free": true,
+    "tags": [
+      "css",
+      "houdini",
+      "support",
+      "reference"
+    ]
+  },
```

Comment:
Wrong URL for the second resource! I guess you just forgot to copy the second one and pasted the same URL twice haha